tests/test_C1_verification.py

```diff
@@ -586,6 +590,152 @@ def test_check_for_label_duplicates_passes():
     ), f"There is no duplicate value for label, but still an error is raised."
 
 
+def test_check_energy_system_can_fulfill_max_demand_sufficient_capacities(caplog):
+    energy_vector = "Electricity"
+    time_series = pd.Series([0, 10, 100, 10, 0])
+    dict_values = {
+        ENERGY_BUSSES: {
+            "Elec": {
+                ENERGY_VECTOR: energy_vector
+            }
+        },
+        ENERGY_CONSUMPTION: {
+            "Demand": {
+                ENERGY_VECTOR: energy_vector,
+                TIMESERIES: time_series
+            }
+        },
+        ENERGY_CONVERSION: {
+            "Inverter": {
+                ENERGY_VECTOR: energy_vector,
+                INSTALLED_CAP: {VALUE: 10}
+            },
+            "Generator": {
+                ENERGY_VECTOR: energy_vector,
+                MAXIMUM_CAP: {VALUE: 90}
+            }
+        }
+    }
+
+    with caplog.at_level(logging.DEBUG):
+        C1.check_energy_system_can_fulfill_max_demand(dict_values)
+    assert (
+        "The check for assets having sufficient capacities to fulfill the maximum demand has successfully passed."
+        in caplog.text
+    ), f"If the maximum/installed capacities of the assets in an energy system are sufficient, a successful debug message is logged."
```

Comment:
```suggestion
    ), f"As maximum/installed capacities of the assets in an energy system are sufficient, a successful debug message should have been logged. This did not happen, with peak generation of {peak_generation} and peak_demand of {peak_demand}."
```
The message here is only printed when the test fails, so it should say what should have happend vs. what happend and give hints how to solve the issue.